config/zones/CA-ON.yaml

```diff
@@ -71,6 +72,10 @@ emissionFactors:
       - datetime: '2021-01-01'
         source: electricityMap, 2021 average
         value: 58.066760035663556
+    nuclear:
+      datetime: '2013-01-01'
+      source: Mallia and Lewis (2013)
```

Comment:
LGTM, but you have to make the source `Mallia and Lewis (2013)` match the one added within sources `Life cycle greenhouse gas emissions of electricity generation in the province of Ontario Canada 2013`

Reply:
I was hoping I could use the authors as a workaround to prevent the detail window from being too wide 😜

But I'll incorporate that change
<img width="1065" alt="image" src="https://user-images.githubusercontent.com/17505337/204856409-d2c36476-8963-471b-995f-10036110ff9f.png">